pkg/ui/src/redux/nodes.ts

```diff
@@ -185,7 +185,7 @@ export function sumNodeStats(
     unavailableRanges: 0,
     replicas: 0,
   };
-  if (_.isArray(nodeStatuses) && _.isObject(livenessStatusByNodeID)) {
+  if (_.isArray(nodeStatuses) && !_.isEmpty(livenessStatusByNodeID)) {
```

Comment:
`livenessStatusByNodeID` is always an object that eiter have or don't have any fields.